src/dapp/CHANGELOG.md

```diff
@@ -10,6 +10,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ### Added
 
 - Support for compilation pipeline going through the Yul intermediate representation
+- Support for installing and running with experimental nix3 commands
```

Comment:
what is nix3?

Reply:
It's the experimental unified CLI, so `nix {build,shell,profile}` etc. Should I make this clearer?

Reply:
ah I see, I didn't know it was called nix3 these days. I guess it could be a little clearer for readers who aren't super up to date with nix terminology?